todo.js

```diff
@@ -1,44 +1,29 @@
-"use strict";
-const todoItems = [];
-
-const id = 0;
+let id = 0;
 
 function getId() {
-  id++;
-  return id;
+    id++;
+    return id;
 }
-function addTodo(newTodo) {
-  const todo = {
-    text,
-    checked: false,
-    id: Date.now(),
-  };
-  todoItems.push(todo);
-  console.log(todoItems);
+
+function addTodo(todos, text) {
+    const todo = {
+        text,
+        checked: false,
+        id: getId(),
+    };
+    return [...todos, todo];
 }
-function deleteTodo(key) {
-  const index = todoItems.findIndex((item) => item.id === Number(key));
```

Comment:
Convert to `Number` shouldn't be used. Because function is supposed to accept right types. If type is invalid it should be considered as error